### PR TITLE
attachment download errors and fix desktop attachment fullscreen view

### PR DESCRIPTION
**Pull request by mmaxim**

### CODE REVIEW

File: shared/chat/conversation/attachment-fullscreen/container.tsx [hunk 1]
```diff
@@ -12,21 +12,27 @@ import {imgMaxWidthRaw} from '../messages/attachment/image/image-render'
...
 const blankMessage = Constants.makeMessageAttachment({})
 
-type OwnProps = Container.RouteProps<{message: Types.MessageAttachment}>
+type OwnProps = Container.RouteProps<{conversationIDKey: Types.ConversationIDKey; ordinal: Types.Ordinal}>
 
 const Connected = (props: OwnProps) => {
-  const [m, setMessage] = React.useState(Container.getRouteProps(props, 'message', blankMessage))
+  const conversationIDKey = Container.getRouteProps(props, 'conversationIDKey', Constants.noConversationIDKey)
+  const inOrdinal = Container.getRouteProps(props, 'ordinal', 0)
+  const state = Container.useSelector(s => s)
```
Comment (chrisnojima) [inline shared/chat/conversation/attachment-fullscreen/container.tsx]: this thrashes every time

Comment (mmaxim) [reply shared/chat/conversation/attachment-fullscreen/container.tsx]: That is existing code, I just moved it.

Comment (mmaxim) [reply shared/chat/conversation/attachment-fullscreen/container.tsx]: https://github.com/keybase/client/blob/master/shared/chat/conversation/attachment-fullscreen/container.tsx#L21

Comment (chrisnojima) [reply shared/chat/conversation/attachment-fullscreen/container.tsx]: sure, its a 1 liner to fix

Comment (mmaxim) [reply shared/chat/conversation/attachment-fullscreen/container.tsx]: It's not, I pass the whole thing to some constants function now. Why does it matter for a fullscreen popup?

**Pull request merged**